### ISSUE TITLE
rgbds v0.3.3 compatibility

### ISSUE THREAD

**Issue by Sanqui** [telefang/bugsite#1]
```Building BugFS filesystem versions/alpha/component/bugfs/directory.bfs
rgblink -n build/bugsite_alpha.sym -m build/bugsite_alpha.map -O baseroms/baserom_alpha.gbc -o build/bugsite_alpha.gbc build/component/bugvm/decode.o build/component/bugvm/optable.o build/component/bugvm/vm_state.o build/component/bugvm/datastack.o build/component/bugvm/predicate.o build/component/windowmanager/contents_config.o build/component/windowmanager/drawframe.o build/component/windowmanager/menu_config.o build/component/windowmanager/print_tools.o build/component/windowmanager/cursor.o build/component/windowmanager/winclr.o build/component/mainscript/window.o build/component/mainscript/scene.o build/component/lcdc/poke.o build/component/lcdc/vmemcopy.o build/component/system/banking.o build/component/bugfs/indexlinkage.o build/component/mainscript/player_name.o build/component/nameinput/string_temporary.o build/component/nameinput/nickname_datablock.o build/versions/alpha/component/bugfs/directory.bugfs.o build/versions/alpha/script/encounters.atbl.o build/versions/beta/script/encounters.atbl.o build/versions/alpha/script/monsters.atbl.o build/versions/beta/script/monsters.atbl.o build/script/chips.atbl.o build/script/keyitems.atbl.o build/script/moves.atbl.o
error: build/versions/alpha/component/bugfs/directory.bugfs.o: Invalid file or object file version [RGB4]
make: *** [Makefile:81: build/bugsite_alpha.gbc] Error 1
```

**Comment by kmeisthax** [comment]
Mmm, sounds like I get to rewrite the RGBDS object structs again. This probably also affects Telefang too. I should have a fix shortly

**Comment by kmeisthax** [comment]
Fixed with the latest commit 3ba7f69809cd756fffdedf4a3281d1b885469549 - you should be able to build with 0.3.3 now. (Earlier versions won't work anymore.)
